package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@theia/che-theia-hosted-plugin-manager-extension",
```

Comment:
hello, why is it @theia if it's a che extension ?

Reply:
will change for `eclipse-che`